### ISSUE TITLE
property-graph-formatter ?

### ISSUE THREAD

**Issue by hchiba1** [g2glab/pg-formatter#1]

**Comment by nichtich** [comment]
I prefer short names and URLs. How about `pg-format`, so the URL becomes <https://g2glab.github.io/pg-format/>? The web application is about **PG format** and not limited to its formatting. It also shows syntax errors, so it's a validator, and it could be extended to simple graph visualization. Another possible name is `pl-playground` similar to [JSON-LD Playground](https://json-ld.org/playground/): a place to try out PG format.

**Comment by ryotayamanaka** [comment]
The short name and the descriptive name, both are reasonable to me. Should we also consider merging this to pg/pgraph repository?

**Comment by hchiba1** [comment]
While I will consider the name `pg-format`,
we will still have a chance to make decisions.

**Comment by nichtich** [comment]
There are three repositories:

- this one: web application to try out, validate, format PG format
- pgraph: CLI and library to parse and convert between property graph formats
- pg: project description of PG format, data model etc.

Two of them could be merged but could also be kept separate. The best location of some files is not fixed anyway. By the way I don't mind moving pgraph into g2lab repository.

**Comment by ryotayamanaka** [comment]
I agree that we should merge them if possible. Creating a new org is an option too. I hope to gradually move forward as far as it is not too urgent!

**Comment by nichtich** [comment]
> Creating a new org is an option too

`pg` and `pgraph` are taken. `pg-format` is available. `property-graphs` or similar is too long.

**Comment by hchiba1** [comment]
(`pgformat`, `pgfmt` and `pg-fmt` are also available.)

Still, I think it is a good idea to invite @nichtich to g2glab organization, and transfer the repository.
